libsolidity/analysis/TypeChecker.cpp

```diff
@@ -3125,8 +3126,7 @@ void TypeChecker::requireLValue(Expression const& _expression, bool _ordinaryAss
 	if (_expression.annotation().isLValue)
 		return;
 
-	return m_errorReporter.typeError(
-		1123_error,_expression.location(), [&]() {
+	return m_errorReporter.typeError(1123_error, _expression.location(), [&]() {
```

Comment:
I think the curly braces go on the beginning of next line.

Reply:
Yes, but we have an exception for lambda functions because this is the transition from the expression to the statement level, and at expression level, we usually but all bracket-like things at the end of the line.